javascript-utility/src/main.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App";
+// import "./index.css";
+// import "./ppcss.css";
+document.body.innerHTML += '<div id="popup-container"></div>';
```

Comment:
change id from `"popup-container"` to something containing `bhashini`.
there sould be other library that might contain `popup-container`.